packages/release/usr-bin.mount.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Binaries
+DefaultDependencies=no
+Conflicts=umount.target
+Before=umount.target
+
+[Mount]
+What=overlay
+Where=PREFIX/bin
+Type=overlay
+Options=lowerdir=PREFIX/fips/bin:PREFIX/bin
```

Comment:
IIUC, in fips variant, we will be shipping the two versions of the binaries? And hide the `no-fips` binaries at runtime?

Reply:
> IIUC, in fips variant, we will be shipping the two versions of the binaries? And hide the no-fips binaries at runtime?

The packages with fips and non-fips binaries are set to conflict with each other, so they won't both be present. A binary like `containerd` will either be in `/usr/fips/bin` or `/usr/bin` depending on the package that's installed, but not in both places.

This overlay mount is to make sure that regardless of what directory they're in, they are available in `/usr/bin` and the standard `PATH`.